packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -1290,6 +1309,8 @@ export interface ChatMessage extends MessageCommon {
     attached?: MessageAttachedStatus;
     // @beta
```

Comment:
Maybe different from this design - but is there a way to merge attachedFilesMetadata and attachments into one property?

Reply:
I think we have discussed before, and it is technically impossible for both `attachedFilesMetadata` and `attachments` to be filled in at the same time. So...its not out of the question. There were just a coupe of things that came to mind when I was working through it:
- What happens if we want to GA inline attachments but keep acs file sharing in beta? Maybe this won't be an issue? 
- Will the names be confusing to developers? Underlying ChatSDK returns the type `ChatAttachment` but we will be mapping it to `FileMetadata`. 
Some things we can chat about.